recipe/meta.yaml

```diff
@@ -1,19 +1,22 @@
-{% set version = "2.8.2" %}
-
+# NOT FOR DEFAULTS.
 package:
   name: python-dateutil
-  version: {{ version }}
+  version: 2.8.3a1
```

Comment:
@mcg1969 Could you please clarify about `python-dateutil' version **2.8.3a1**? You uploaded it by this PR https://github.com/AnacondaRecipes/python-dateutil-feedstock/pull/3 and now we need to add python 3.11 support for that version

Reply:
We had to create a "special" version for Snowflake a while ago, which had a specific patch that was required for their use case (so no such version exists upstream, we just made it for a customer). Ordinarily we wouldn't keep upgrading/maintaining such a version, as we typically incorporate newer patches into the last released upstream version. In this case, however, the patch is probably incomplete and/or not fully tested, so shipping it with 2.8.2 wouldn't be right either, unless we go the extra mile and finish up the patch (for which we lack resources at the moment). So yes, I'd say for now we have to keep this version around just for Snowflake.
(N.B.: To make matters worse, the patch is probably abandoned).

Reply:
The Python packaging ecosystem and conda support what's called a local version identifier (https://peps.python.org/pep-0440/#local-version-identifiers). So basically instead of `2.8.3a1`, we could use `2.8.2+snowflake<n>` or something like that.

Reply:
I agree `2.8.2+snowflake<n>` would be a better solution. But now I'm confused about how to resolve the version number for the existing **v2.8.3a1** because if we build `2.8.2+snowflake<n>_<build_number>` then the conda solver will pick v2.8.3a1 up as it has a greater number.